packages/effection/test/mailbox.test.ts

```diff
@@ -181,11 +181,11 @@ describe("Mailbox", () => {
       spawn(function*() {
         yield source.pipe(destination);
         yield;
-      });
+      } as OperationFn<unknown>);
```

Comment:
Let’s use operation where we can. OperationFn is also an Operation and it will be deprecated.

Also, Return type is not unknown, it is void